release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -132,6 +132,11 @@ You can now configure user-defined labels and tags in Google Cloud Platform (GCP
 
 In {product-title} 4.14, you can install a cluster on Microsoft Azure in a restricted network for installer-provisioned infrastructure (IPI) and user-provisioned infrastructure (UPI). For IPI, you can create an internal mirror of the installation release content on an existing Azure Virtual Network (VNet). For UPI, you can install a cluster on Microsoft Azure by using infrastructure that you provide. For more information, see xref:../installing/installing_azure/installing-restricted-networks-azure-installer-provisioned.adoc#installing-restricted-networks-azure-installer-provisioned[Installing a cluster on Azure in a restricted network] and xref:../installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc#installing-restricted-networks-azure-user-provisioned[Installing a cluster on Azure in a restricted network with user-provisioned infrastructure].
 
+[id="ocp-4-14-bare-metal-installation-disk-id"]
+==== Specifying installation disks using by-path device alias during bare metal installations
+
+You can now specify the installation disk using a by-path device alias, such as `deviceName: "/dev/disk/by-path/pci-0000:01:00.0-scsi-0:0:0:0"`, when you install a cluster on bare metal with installer-provisioned infrastructure. This type of disk alias persists across reboots. For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-the-install-config-file_ipi-install-installation-workflow[Configuring the install-config.yaml file].
```

Comment:
Note that this works on both IPI and the agent-based installer (different bugs, but both were added in 4.14 as well as having been backported). Link for the agent-based installer: https://docs.openshift.com/container-platform/4.13/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#root-device-hints_preparing-to-install-with-agent-based-installer

Reply:
Thanks, I added mention of the Agent based installer